src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,199 @@
+package duke;
+
+import duke.command.Parser;
+import duke.command.Storage;
+import duke.command.TaskList;
+import duke.command.Ui;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+import duke.exception.MissingContentException;
+import duke.exception.DukeException;
+
+
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+
+enum action {
+    Bye,
+    List,
+    Mark,
+    Unmark,
+    Todo,
+    Deadline,
+    Event,
+    Delete,
+    Find
+}
```

Comment:
Nice use of enum! Class/enum names are recommended to be nouns and written in PascalCase (e.g. Action instead of action)